scripts/gen-lvms-kustomize.sh

```diff
@@ -0,0 +1,184 @@
+#!/bin/bash
+
+# expect that the common.sh is in the same dir as the calling script
+SCRIPTPATH="$( cd "$(dirname "$0")" >/dev/null 2>&1 ; pwd -P )"
+. "${SCRIPTPATH}"/common.sh --source-only
+
+# Some defaults
+DISK_SIZE=${DISK_SIZE:-50}
```

Comment:
Do you want to default to 100?

Reply:
on L24 we have `DEVICES["vdb"]=100`: that should be enough but I'm ok to change this value as well to reflect the minimum requirement for a disk.
I'll do that in my next PS

Reply:
Done